test/iron-menu-behavior.html

```diff
@@ -139,107 +139,129 @@
+            Polymer.Base.async(function() {
+              var ownerRoot = Polymer.dom(menu.firstElementChild).getOwnerRoot() || document;
+              var activeElement = Polymer.dom(ownerRoot).activeElement;
+              assert.equal(activeElement, menu.firstElementChild, 'menu.firstElementChild is focused');
+              done();
+            });
           });
         });
 
         test('first item gets focus when menu is focused in a single item menu', function(done) {
           var menu = fixture('single-item');
-          MockInteractions.focus(menu);
-          Polymer.Base.async(function() {
-            var ownerRoot = Polymer.dom(menu.firstElementChild).getOwnerRoot() || document;
-            var activeElement = Polymer.dom(ownerRoot).activeElement;
-            assert.equal(activeElement, menu.firstElementChild, 'menu.firstElementChild is focused');
-            done();
+          waitForIronItemsChanged(menu, function() {
+            MockInteractions.focus(menu);
+            Polymer.Base.async(function() {
+              var ownerRoot = Polymer.dom(menu.firstElementChild).getOwnerRoot() || document;
+              var activeElement = Polymer.dom(ownerRoot).activeElement;
+              assert.equal(activeElement, menu.firstElementChild, 'menu.firstElementChild is focused');
+              done();
+            });
           });
         });
 
         test('selected item gets focus when menu is focused', function(done) {
           var menu = fixture('basic');
-          menu.selected = 1;
-          MockInteractions.focus(menu);
-          Polymer.Base.async(function() {
-            var ownerRoot = Polymer.dom(menu.selectedItem).getOwnerRoot() || document;
-            var activeElement = Polymer.dom(ownerRoot).activeElement;
-            assert.equal(activeElement, menu.selectedItem, 'menu.selectedItem is focused');
-            done();
+          waitForIronItemsChanged(menu, function() {
+            menu.selected = 1;
+            MockInteractions.focus(menu);
+            Polymer.Base.async(function() {
+              var ownerRoot = Polymer.dom(menu.selectedItem).getOwnerRoot() || document;
+              var activeElement = Polymer.dom(ownerRoot).activeElement;
+              assert.equal(activeElement, menu.selectedItem, 'menu.selectedItem is focused');
+              done();
+            });
           });
         });
 
         test('focusing on next item skips disabled items', function(done) {
           var menu = fixture('disabled');
-          MockInteractions.focus(menu);
-          // Wait for async focus
-          Polymer.Base.async(function() {
-            // Key press down
-            MockInteractions.keyDownOn(menu, 40);
-
+          waitForIronItemsChanged(menu, function() {
+            MockInteractions.focus(menu);
+            // Wait for async focus
             Polymer.Base.async(function() {
-              var focusedItem = Polymer.dom(menu).node.focusedItem;
```

Comment:
`Polymer.dom(x).node` returns `x` [in 2.x](https://github.com/Polymer/polymer/blob/master/lib/legacy/polymer.dom.html#L42) and returns a wrapper around `x` in 1.x, but these tests never used `focusedItem` to manipulate the DOM, so I removed it. I don't really know what the original intent was but I don't think there's any harm in putting it back if that's preferable.